middleware/security.py

```diff
@@ -34,12 +37,7 @@ def decorator(*args, **kwargs):
         if request.headers and 'Authorization' in request.headers:
             authorization_header = request.headers['Authorization'].split(" ")
             if len(authorization_header) >= 2 and authorization_header[0] == "Bearer":
-                try:
-                    token = request.headers['Authorization'].split(" ")[1]
-                    payload = jwt.decode(token, os.getenv('SECRET_KEY'), algorithms=['HS256'])
-                    api_key = payload['api_key']
-                except jwt.InvalidTokenError:
-                    api_key = request.headers['Authorization'].split(" ")[1]
```

Comment:
If we're removing JWT here, we need to account for it in the login function in User.py. Right now, we generate an API key but encode it with JWT first before sending it to the frontend. I think we can just create the API key and store it for now. At the very least, we should cut the lines that encode it with JWT and send this false token to the frontend that we can't handle anymore